docs/adr/adr-006-fraud-service.md

```diff
@@ -62,157 +64,169 @@ In addition, `das.Daser`:
 
 1. Creates a BEFP:
 
-```go
```

Comment:
The following sections were indented so that they display under the ordered list item they are relevant to.